indexer/services/ender/__tests__/handlers/markets/market-create-handler.test.ts

```diff
@@ -137,13 +137,9 @@ describe('marketCreateHandler', () => {
     ) as MarketFromDatabase;
     expect(market.minPriceChangePpm).toEqual(50);
 
-    expect(loggerError).toHaveBeenCalledWith(expect.objectContaining({
-      at: 'MarketCreateHandler#logAndThrowParseMessageError',
-      message: 'Market in MarketCreate already exists',
-    }));
     expect(loggerCrit).toHaveBeenCalledWith(expect.objectContaining({
-      at: 'onMessage#onMessage',
-      message: 'Error: Unable to parse message, this must be due to a bug in V4 node',
+      at: expect.stringContaining('PL/pgSQL function dydx_market_create_handler('),
```

Comment:
Note that the main log location will now also include the stack trace from postgres which contains function names (and now correct line numbers)